### PR TITLE
graph homework

### PR DESCRIPTION
**Pull request by NMandel527**
Screenshot 2024-02-28 141106

### CODE REVIEW

**Comment by andrewoid** [comment]
Include a screenshot in the description field of the PR.

File: src/main/java/mandel/projectile/ProjectileGraph.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+{
+    private Projectile projectile = new Projectile(0, 0);
+
+    @Override
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+
+        g.translate(0, getHeight());
+        g.setColor(Color.BLUE);
+        g.fillOval((int) projectile.getInterceptX() / 2, (int) (-projectile.getPeakY()), 10, 10);
```
Comment (andrewoid) [inline src/main/java/mandel/projectile/ProjectileGraph.java:16]: the oval needs to be centered on the curve. Yours is not.

**Comment by NMandel527** [comment]
Screenshot 2024-02-27 233111

File: src/main/java/mandel/projectile/ProjectileGraph.java (hunk 1 above)
Comment (NMandel527) [reply src/main/java/mandel/projectile/ProjectileGraph.java]: Is it properly centered now?

File: src/main/java/mandel/projectile/ProjectileGraph.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+        g.translate(0, getHeight());
+        g.setColor(Color.BLUE);
+        g.fillOval((int) (projectile.getInterceptX() / 2) - 5, (int) (-projectile.getPeakY()) - 5, 10, 10);
+
+        int x;
+        int y;
+
+        g.setColor(Color.BLACK);
+
```
Comment (andrewoid) [inline src/main/java/mandel/projectile/ProjectileGraph.java:22]: projectile.setSeconds(0);

Otherwise the for loop can have strange results.

Comment (NMandel527) [reply src/main/java/mandel/projectile/ProjectileGraph.java:23]: Oh ok thank you, I just fixed it. Should I upload another screenshot?

Comment (andrewoid) [reply src/main/java/mandel/projectile/ProjectileGraph.java:23]: Yes.

**Comment by NMandel527** [comment]
Screenshot 2024-02-28 141106

**Review by andrewoid — APPROVED** [review]

**Pull request merged**